onnxruntime/python/tools/transformers/quantize_helper.py

```diff
@@ -68,6 +68,7 @@ def quantize_onnx_model(onnx_model_path, quantized_model_path, use_external_data
 
         if use_external_data_format:
             from pathlib import Path
+            Path(os.path.split(quantized_model_path)[0]).mkdir(parents=True, exist_ok=True)
```

Comment:
How about
```
Path(quantized_model_path).parent.mkdir(parents=True, exist_ok=True)
```

Reply:
That was my first attempt but I got an error parent has no mkdir error during debugging. Must have been a red herring as this works during a full run of convert_to_onnx.py